.env.sample

```diff
@@ -17,3 +17,5 @@ WEBPACK_PORT="3808"
 DB_USERNAME=
 DB_PASSWORD=
 DB_HOST=
+PRESET_CLOUD=
+CLOUD_NAME=
```

Comment:
Can we extract this from CLOUDINARY_URL?

Reply:
how you say it? Example, do a regular expression and extract information? this way is good?

Reply:
```ruby
Cloudinary.config.cloud_name
```